crates/bevy_window/src/window.rs

```diff
@@ -507,10 +544,32 @@ impl Window {
         self.command_queue.drain(..)
     }
 
+    #[inline]
+    pub fn command_queue(&self) -> &[WindowCommand] {
+        &self.command_queue
+    }
```

Comment:
i'm not a big fan of adding this function, the command queue feels like an internal implementation detail that shouldn't be exposed.

Reply:
Exposing the whole queue is unnecessary, because I only really care about one command (or anything that indicates that the icon path has been changed). 
I need some way to only react to every new set_icon call (with a path) once, right?